mod-source-record-storage-server/src/main/java/org/folio/consumers/ParsedRecordChunksKafkaHandler.java

```diff
@@ -7,25 +7,15 @@
 import io.vertx.kafka.client.consumer.KafkaConsumerRecord;
 import io.vertx.kafka.client.producer.KafkaHeader;
 import io.vertx.kafka.client.producer.KafkaProducer;
-import io.vertx.kafka.client.producer.KafkaProducerRecord;
-
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-
 import org.folio.DataImportEventPayload;
 import org.folio.dao.util.ParsedRecordDaoUtil;
 import org.folio.dataimport.util.OkapiConnectionParams;
-import org.folio.kafka.AsyncRecordHandler;
-import org.folio.kafka.KafkaConfig;
-import org.folio.kafka.KafkaHeaderUtils;
-import org.folio.kafka.KafkaTopicNameHelper;
-import org.folio.rest.jaxrs.model.Event;
-import org.folio.rest.jaxrs.model.EventMetadata;
-import org.folio.rest.jaxrs.model.Metadata;
-import org.folio.rest.jaxrs.model.RecordCollection;
-import org.folio.rest.jaxrs.model.RecordsBatchResponse;
+import org.folio.kafka.*;
```

Comment:
please consider not using wildcard imports

Reply:
I thought, that I removed it with explicit declaration, thanks for noticing it)